api/get_rbg_stream_site.php

```diff
@@ -1,26 +1,29 @@
 <?php
 include 'simple_html_dom.php';
-include_once 'CacheAccess.php';
 
 function ParseInformation($link) {
-    // First get html of remote page
-    $html = CacheAccess::getHtml($link);
+    $CACHEKEY = "get_rbg_stream_site_" . $link;
+    if (apcu_exists($CACHEKEY)) {
+        return apcu_fetch($CACHEKEY);
+    } else {
+        // First get html of remote page
+        $html = file_get_html($link);
 
-    $veranstaltung = $html->find("main", 1)->find("font", 0)->find("b", 0)->innertext;
-    $date = $html->find("main", 1)->find("font", 0)->find("b", 1)->innertext;
-    $time = $html->find("main", 1)->find("font", 0)->find("b", 2)->innertext;
+        $veranstaltung = $html->find("main", 1)->find("font", 0)->find("b", 0)->innertext;
+        $date = $html->find("main", 1)->find("font", 0)->find("b", 1)->innertext;
+        $time = $html->find("main", 1)->find("font", 0)->find("b", 2)->innertext;
```

Comment:
What happens to these three function calls above if $html isn't an object? Because you do a `if ($html)` below, but its not really clear to my, why the above are outside of that condition?